syncano_cli/commands.py

```diff
@@ -2,37 +2,39 @@
 import os
 from getpass import getpass
 
-import six
+import click
 import syncano
 from syncano.exceptions import SyncanoException
-from syncano_cli.commands_base import CommandContainer
-from syncano_cli.parse_to_syncano.commands import Configure, Parse  # noqa
-from syncano_cli.sync.commands import Pull, Push  # noqa
+from syncano_cli.config import ACCOUNT_CONFIG, ACCOUNT_CONFIG_PATH
 
-COMMAND_NAMESPACE = 'toplevel'
 
+@click.group()
+@click.pass_context
+def top_level(context, config):
```

Comment:
Probably I not figured it out how to nicely handle both - context and config as a one :) Will take a closer look, you know this refactor becomes something bigger than expected and I've lost some context on the way ;)